Production/crab/config/2018/DY_embedded.txt

```diff
@@ -1,4 +1,4 @@
-globalTag=102X_upgrade2018_realistic_v19 sampleType=Emb_18ABC runSVfit=False runKinFit=False storeLHEinfo=False saveGenParticleInfo=False isEmbedded=True
+globalTag=102X_dataRun2_v11 sampleType=Emb_18ABC runSVfit=False runKinFit=False storeLHEinfo=False saveGenParticleInfo=False isEmbedded=True
```

Comment:
For 2018, embedded should be split depending on the period (same as for the data)